config/config.toml

```diff
@@ -9,7 +9,7 @@ port = 9000
 # "static" - no discovery, only connect to static peers
 # "discv5" - Enable discovery v5
 discovery = "static"
-bootnodes = ""
+#bootnodes = ""
```

Comment:
Probably should just be 
```suggestion
bootnodes = []
```

Reply:
I tried that, it didn't work :/

Reply:
hmm, it did for Cem and I.  I just use a custom config file so forgot to update the checked in version.

Reply:
it really should be defaulted in ArtemisConfiguration so that removal doesn't tank everyones instances...